packages/runtime/src/context.ts

```diff
@@ -71,6 +71,10 @@ export type Context = {
     content: Content,
     options?: UpsertOptions,
   ) => Promise<ModelInstanceDocument<Content> | null>
+  /**
+   * Enable indexing for an existing document.
+   */
+  enableDocIndexing: (model: string, opts?: UpdateOpts) => Promise<void>
```

Comment:
I think it's confusing to name the stream ID `model` here as it's the stream that gets updated, nothing to do with the model

```suggestion
  enableDocIndexing: (id: string, opts?: UpdateOpts) => Promise<void>
```